src/derivatives/atlas.md

```diff
@@ -0,0 +1,553 @@
+only one data type (such as `anat/`) is stored under the `tpl-<label>/` directory.
+
+The [`cohort-<label>` directory and entity](../appendices/entities.md#cohort) MUST be specified for templates
+with several cohorts.
+
+!!! note "Dataset structure"
+
+    In templates/atlases the [`tpl-<label>`](../appendices/entities.md#tpl) and
+    [`cohort-<label>`](../appendices/entities.md#cohort) entities are structurally
+    analogous to the [`sub-<label>`](../appendices/entities.md#sub)
+    and [`ses-<label>`](../appendices/entities.md#ses) entities,
+    appearing as both directories and entities.
+    The [`cohort-<label>`](../appendices/entities.md#cohort) entity is used
+    to track mappings of arbitrary subject/session pairs.
+
+In BIDS, a template is considered *any* aggregation of data and the `tpl-` entity
+replaces the subject-level `sub-` entity when aggregating data across subjects.
+The `tpl-` entity and `sub-` entity are mutually exclusive in a given file.
+When the `tpl-` entity is used without the `atlas-` entity, as in the following examples,
+the imaging data serves as an instantiation of a space.
+When the `tpl-` entity is used in conjunction with the `atlas-` entity,
+the intention is to describe aggregations of data in a particular space,
+as described in the [Derived atlases](#derived-atlases) section, below.
+For uses of the `atlas-` entity without the `tpl-` entity,
+see [Imaging derivatives - Derivatives from atlases](imaging.md#derivatives-from-atlases).
+
+### Example: Single-cohort template
+
+For the pipeline that generated [`MNI152NLin2009cAsym`](../appendices/coordinate-systems.md#standard-template-identifiers),
+the outputs could look like the following example:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "mni152nlin2009casym-pipeline": {
+      "tpl-MNI152NLin2009cAsym": {
+         "tpl-MNI152NLin2009cAsym_res-1_T1w.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-1_T1w.json": "",
+         "tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.json": "",
+         "tpl-MNI152NLin2009cAsym_res-1_probseg.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-1_probseg.json": "",
+         "tpl-MNI152NLin2009cAsym_res-2_T1w.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-2_T1w.json": "",
+         "tpl-MNI152NLin2009cAsym_res-2_label-brain_mask.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-2_label-brain_mask.json": "",
+         "tpl-MNI152NLin2009cAsym_res-2_probseg.nii.gz": "",
+         "tpl-MNI152NLin2009cAsym_res-2_probseg.json": "",
+      },
+   }
+})
+}}
+
+### Example: Multi-cohort template
+
+In the case that the template-generating pipeline derives
+several cohorts, the file structure must employ the
```

Comment:
```suggestion
results for several cohorts, the file structure must employ the
```

since I bet `cohorts` are typically assembled by conscious decisions of researchers (e.g. based on age etc), and not pipelines (like e.g. some unsupervised clustering).